src/widgets/menu/menu.tsx

```diff
@@ -25,43 +30,43 @@ export type MenuCSSClasses = {
   /**
    * CSS class to add to the root element.
    */
-  root: string;
+  root?: string;
```

Comment:
```suggestion
  root?: string | string[];
```

double-check if there are others in this (wrong) style please

Reply:
This seems to be all of them: https://github.com/algolia/instantsearch.js/pull/4774/commits/0f2a8aad54e7185faabfb22e2c8b2bfca3bc971a